<ui version="4.0" >
 <class>prefsWnd</class>
 <widget class="QDialog" name="prefsWnd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Video Recorder</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QCheckBox" name="ckEnableProxy" >
     <property name="text" >
      <string>Enable Proxy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProxy" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Proxy Settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>&amp;Host</string>
          </property>
          <property name="buddy" >
           <cstring>leProxyHost</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProxyHost" >
          <property name="text" >
           <string>p-niceway</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>252</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>&amp;Port</string>
          </property>
          <property name="buddy" >
           <cstring>sbProxyPort</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbProxyPort" >
          <property name="maximum" >
           <number>65535</number>
          </property>
          <property name="value" >
           <number>3128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>&amp;User</string>
          </property>
          <property name="buddy" >
           <cstring>leProxyUser</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProxyUser" />
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>&amp;Password</string>
          </property>
          <property name="buddy" >
           <cstring>leProxyPassword</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProxyPassword" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Regular expression for video link</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leLinkRE" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Regular expression for video list</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leListRE" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Default url</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leDefaultUrl" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Max retries</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="sbRetries" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Minimum size</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="sbMinSize" >
       <property name="maximum" >
        <number>100000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbObstinate" >
     <property name="text" >
      <string>Obstinate donwload when programmed</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
